packages/notebook/src/windowing.ts

```diff
@@ -97,6 +99,30 @@ export class NotebookWindowedLayout extends WindowedLayout {
     }
   }
 
+  /**
+   * Returns Notebook's footer
+   *
+   * @return NotebookFooter | null
```

Comment:
This is an oversight on my side on the header documentation. But actually getter+setter should only have a single doc string.

```suggestion
   * Notebook widget's footer
```